ui/.gitignore

```diff
@@ -0,0 +1,23 @@
+.DS_Store
+node_modules
+/dist
+
+
+# local env files
+.env.local
+.env.*.local
+
+# Log files
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
+pnpm-debug.log*
+
+# Editor directories and files
+.idea
```

Comment:
Please use [global gitignore](https://gist.github.com/subfuzion/db7f57fff2fb6998a16c) for private tool artifacts

Reply:
Done, good idea